xmlpull/.gitignore

```diff
@@ -0,0 +1,16 @@
+bin
+build
+SCRATCH
```

Comment:
(nitpick) do wee still need SCRATCH etc ignored?

Reply:
Likely not, this would be a leftover from the Ant build. Will check and remote those entries.